make.js

```diff
@@ -405,7 +405,7 @@ target.test = function() {
 
 target.testLegacy = function() {
     ensureTool('tsc', '--version', 'Version 2.3.4');
-    ensureTool('mocha', '--version', '5.2.0');
+    ensureTool('mocha', '--version', '6.2.3');
```

Comment:
Any idea why this might have been necessary?

Reply:
fix was merged in #16378